### ISSUE TITLE
[feature request] Left-handed mouse buttons

### ISSUE THREAD

**Issue by toimine** [window-maker/wmaker#45]
This is an option that exists in at least Xfce and Windows that swaps the left and right mouse buttons, making you press buttons with the right button, spawn context menus with the left button etc. Any games will also treat your right button as a left button, and vice versa. 
I just tried setting up Window Maker on NixOS 23.11, but could find an option for it, maybe I missed something?

**Comment by d-torrance** [comment]
I think this feature is probably beyond the scope of a window manager, but you could try adding the following to `~/GNUstep/Library/WindowMaker/autostart`:

```
xmodmap -e "pointer = 3 2 1"
```
Note that `autostart` should be executable for this to work.

**Comment by toimine** [comment]
Of course, xmodmap! I forgot I could use that. That works perfectly, thanks.

**Comment by gryf** [comment]
the other option is to configure Xorg for input device, i.e.:
```
Section "InputDevice"
    Identifier  "Mouse"
    Driver      "mouse"
    Option      "Device" "/dev/input/mouse0"
    Option      "ButtonMapping" "3 2 1 4 5"
EndSection
```
although xmodmap way feels more flexible.
